core/domain/html_validation_service.py

```diff
@@ -705,6 +705,23 @@ def is_parsable_as_xml(xml_string):
         return False
 
 
+def convert_svg_diagram_to_image_for_soup(soup_context):
+    """"Renames oppia-noninteractive-svgdiagram tag to
+    oppia-noninteractive-image and changes corresponding attributes for a given
+    soup context
+
+    Args:
+        soup_context: soup. The bs4 soup context.
```

Comment:
Is "soup" a valid type? I don't see it in other docstrings.

Reply:
Ah, I've changed it to `bs4.BeautifulSoup` now, we use that in `validate_soup_for_rte`'s docstring.